index.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./assets/css/style.css" />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.0.8/css/all.css"
+      integrity="sha384-3AB7yXWz4OeoZcPbieVW64vVXEwADiYyAEhwilzWsLw+9FgqpyjjStpPnpBO8o8S"
+      crossorigin="anonymous"
+    />
+    <title>Welcome to Sumaia's Portfolio</title>
+  </head>
+
+  <body>
+    <!-- hearder section -->
+    <header>
+      <nav class="nav-container">
+        <!-- for portfolio heading -->
+
+        <h1>portfo<span>lio</span></h1>
+
+        <div class="nav-items">
+          <!-- div containing nav links -->
+          <div class="nav-link"><a href="#About-me">About Me</a></div>
+          <div class="nav-link"><a href="#projects">Projects</a></div>
+          <div class="nav-link"><a href="#Contact-me">Contact Me</a></div>
+        </div>
+      </nav>
+    </header>
+    <!-- main section -->
+    <main>
+      <!-- added id for the about me section -->
+      <section class="flex-container" id="About-me">
+        <div class="flex-img">
+          <img
+            src="./assets/images/sumaia-image.jpg"
+            alt="headshot of sumaia"
+            class="profile-pic"
+          />
+        </div>
+        <!-- about me section -->
+        <div class="flex-text">
+          <h1>Sum<span>aia</span> sala<span>uddin</span></h1>
+          <p>
+            Sumaia was a marketing graduate but has been gradually retraining
+            herself for the digital world. She is an enthusiastic individual and
+            eager to work with a dynamic team of web developers.Currently she is
+            working as a Digital Marketing Assistant at Favorite Food Ltd.
+          </p>
+          <h2>Skills:</h2>
+          <p>HTML | CSS | JavaScript</p>
+        </div>
+      </section>
+
+      <!-- projects -->
+
+      <section id="projects">
+        <h2>Projects</h2>
+
+        <!-- main project with the clikable link -->
+
+        <div class="project-container-main">
+          <div class="project-image">
+            <a
+              target="_blank"
+              href="https://sumaiasorna.github.io/semantic-html-code-refactor/"
+              ><img
+                src="./assets/images/Horiseon-marketing-website.png"
+                alt="Marketing webpage image"
+            /></a>
+          </div>
+          <div class="project-description">
+            <h3>Refactor Code</h3>
+            <p>
+              Refactoring Horiseon-marketing-website with semantic html for
+              improved accessibility.
+            </p>
+          </div>
+        </div>
+        <!-- six projects in a flex container-->
+        <div class="cards-container">
+          <div class="small-project-card">
+            href="">
```

Comment:
This line is showing as text on your portfolio, please delete

Reply:
Thank you chelsea